cpp/community_detection_module/CMakeLists.txt

```diff
@@ -7,25 +7,11 @@ set(community_detection_module_src
     algorithm/louvain.cpp)
 
 add_subdirectory(grappolo)
-add_query_module(community_detection 1 "${community_detection_module_src}")
 
+add_query_module(community_detection 1 "${community_detection_module_src}")
 target_include_directories(community_detection PRIVATE ${GRAPPOLO_HEADERS} ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(community_detection PRIVATE mg_utility basic_cd full_syn_opt coloring util OpenMP::OpenMP_CXX)
 
-# Module tests
-if (NOT MAGE_CUGRAPH_ENABLE)
-    include(GoogleTest)
-    set(community_detection_test_src
-        community_detection_test.cpp
-        algorithm/louvain.cpp)
-
-    add_executable(community_detection_test "${community_detection_test_src}")
-    target_include_directories(community_detection_test PRIVATE ${GRAPPOLO_HEADERS} ${CMAKE_CURRENT_SOURCE_DIR})
-    target_link_libraries(community_detection_test PRIVATE mg_utility basic_cd full_syn_opt coloring util OpenMP::OpenMP_CXX mage_gtest)
-    gtest_add_tests(TARGET community_detection_test)
-endif()
-
-
```

Comment:
These tests were ported to e2e tests.
The tests were not usable because now community_detection library uses mgp_graph and the graph testing library doesn't use that.